### PR TITLE
Tests for register and index

### PR DESCRIPTION
**Pull request by starbr3aker**
Register test has tests to ensure valid inputs
Index test sends user to root after faulty login

### CODE REVIEW

File: test/index_test.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        
+    })
+
+    after(async () => {
+        agent.close();
+        await User.find({'username':inauth_user.username}).remove()
+        await User.find({'username':dummy_user.username}).remove()
+    })
+
+    it('un authenticated user is redirected to /', async() =>{
```
Comment (shreyasseshadri) [inline test/index_test.js]: This test is already present in the [init](https://github.com/shreyasseshadri/chatsapp/blob/ebd014202cd66fcdf87e73b2d5fc31d4f15ad340/test/init.js#L48) test...I don't think there is a need for a seperate file just to check this

Comment (starbr3aker) [reply test/index_test.js]: I didn't notice, I'll remove it.

File: test/register_test.js [hunk 2]
```diff
@@ -20,9 +20,51 @@ describe('Register tests',  () => {
...
     })
 
-    it('Checking phone number length', async () => {
+    it('Checking phone number length not less than 10', async () => {
         const register_res = await agent.post('/register').send({phone: '657488907', username:'stlayer',password:'PaSsW0rd!', name:"Gleep glopb"});
         expect(register_res).to.have.status(400); 
     });
+    it('Checking phone number length not more than 10', async () => {
+        const register_res = await agent.post('/register').send({phone: '65748890721', username:'stlayer',password:'PaSsW0rd!', name:"Gleep glopb"});
+        expect(register_res).to.have.status(400);
```
Comment (shreyasseshadri) [inline test/register_test.js:37]: Please verify the error message along with the status for all the cases

Comment (starbr3aker) [reply test/register_test.js:37]: I changed the phone number in all cases, tests work now

Comment (shreyasseshadri) [reply test/register_test.js:37]: I meant the server is returning a message along with the status right ? Verify that as well in `register_res`

Comment (starbr3aker) [reply test/register_test.js:37]: Added message verification

File: test/message_test.js [hunk 3]
```diff
@@ -87,11 +87,10 @@ describe('Message Tests',  () => {
         agent1 = chai.request.agent(server);
         agent2 = chai.request.agent(server);
 
-
```
Comment (shreyasseshadri) [inline test/message_test.js:90]: Don't add or remove spaces

File: test/message_test.js [hunk 4]
```diff
@@ -87,11 +87,10 @@ describe('Message Tests',  () => {
...
         const resp1 = await agent1.post('/register').send(dummy_user1);
         console.log(resp1.text)
         expect(resp1).to.have.status(200)
-
+
```
Comment (shreyasseshadri) [inline test/message_test.js:94]: Dont add or remove spaces

**Pull request merged**